.ci/scripts/build_all_plugins.sh

```diff
@@ -3,6 +3,11 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+arg="$1"
 cd all_packages
-flutter build windows --debug
-flutter build windows --release
+if [[ "$arg" == "web" ]]; then
```

Comment:
This is skipping everything on web, rather than just skipping `debug`.

Reply:
Good catch! With changes to the other comment, this is resolved and `web` case will be handled with a comment for the debug mode.